src/pages/community/meetups.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Druid Community Meetups
+layout: simple_page
+canonical: 'https://druid.apache.org/community/meetups'
+---
+
+## Druid Community Meetups
+
+Check out [Apache Druid on meetup.com](https://www.meetup.com/topics/apache-druid/) for links to meetups in cities all over the world.
+
+All Apache Druid Meetups subject to the Apache [anti-harassment policy](https://www.apache.org/foundation/marks/events#anti-harassment) and speakers are expected to follow the guidelines below.
+
+### Speaker Guidelines
+
+**Skip the Sales Pitch:** Keep your talk unbiased and educational. It’s okay to mention commercial products you’re using, but only if they are a key part of your data architecture and Apache Druid or if what you are demonstrating can be accomplished with a free or free tier version of the commercial product or service. Content which is solely focused on promotion of a product or service is never permitted at an Apache Druid Meetup.
+
+**Share Challenges:**  Feel good about sharing roadblocks you hit and how you worked around them – or ones you might still be struggling with – there may be a community member in the audience who can help you out! Engineering always involves trade-offs, so being realistic about them helps establish you as a credible advisor.
+
+**Focus on the Technical:** This audience understands and has built data architectures – if you get it, they’ll get it. Of course, start with the basics of who you are, what you built, and why before going all-in on the technical insights, examples, and demos.
+
+**Keep an open mind:** The Apache Druid project has [a vendor neutrality policy](https://community.apache.org/projectIndependence#:~:text=Apache%20projects%20are%20managed%20independently) because it is important to keep an open mind about approaches and not only share one product or solution when many are available. A community that obviously favors one specific vendor in some exclusive way will often discourage new contributors from competing vendors, and this would be an issue for the long-term health of the project.
```

Comment:
```suggestion
**Keep an open mind:** The Apache Druid project has [a vendor neutrality policy](https://community.apache.org/projectIndependence#:~:text=Apache%20projects%20are%20managed%20independently) because it is important to keep an open mind about different ways to solve problems. A community that obviously favors one specific vendor in some exclusive may discourage new contributors from competing vendors, potentially impacting the long-term health of the project.
```